### PR TITLE
UCP/CUDA: Use static_bitmap structure

### PR DESCRIPTION
**Pull request by yosefe**
## Why
Switch from previous to new static bitmap implementation, to allow code cleanup

### CODE REVIEW

**Review by iyastreb — APPROVED** [review]

File: src/ucp/core/ucp_context.h [hunk 1]
```diff
@@ -20,7 +20,6 @@
 #include <uct/api/v2/uct_v2.h>
 #include <ucs/datastruct/mpool.h>
 #include <ucs/datastruct/queue_types.h>
-#include <ucs/datastruct/bitmap.h>
```
Comment (tvegas1) [inline src/ucp/core/ucp_context.h:23]: seems while at it we could also add:
```diff
diff --git a/src/uct/ib/dc/dc_mlx5.h b/src/uct/ib/dc/dc_mlx5.h
index 4c2c0f09f..877a73787 100644
--- a/src/uct/ib/dc/dc_mlx5.h
+++ b/src/uct/ib/dc/dc_mlx5.h
@@ -14,7 +14,6 @@
 #include <uct/ib/ud/base/ud_iface_common.h>
 #include <uct/ib/ud/accel/ud_mlx5_common.h>
 #include <ucs/debug/assert.h>
-#include <ucs/datastruct/bitmap.h>


 /*
diff --git a/src/uct/ib/rdmacm/rdmacm_cm.h b/src/uct/ib/rdmacm/rdmacm_cm.h
index 5d1a326b6..300cc2325 100644
--- a/src/uct/ib/rdmacm/rdmacm_cm.h
+++ b/src/uct/ib/rdmacm/rdmacm_cm.h
@@ -11,7 +11,6 @@
 #include <ucs/datastruct/khash.h>
 #include <ucs/sys/string.h>
 #include <ucs/type/spinlock.h>
-#include <ucs/datastruct/bitmap.h>
 #if HAVE_DEVX
 #include <uct/ib/mlx5/ib_mlx5.h>
 #endif
```

File: src/uct/cuda/cuda_copy/cuda_copy_iface.h [hunk 2]
```diff
@@ -77,7 +77,7 @@ typedef struct uct_cuda_copy_iface {
         uct_async_event_cb_t    event_cb;
     } async;
 
-    /* 2D bitmap representing which streams in queue_desc matrix 
+    /* 2D bitmap representing which streams in queue_desc matrix
```
Comment (brminich) [inline src/uct/cuda/cuda_copy/cuda_copy_iface.h:80]: unrelated

**Review by tvegas1 — APPROVED** [review]

**Review by brminich — APPROVED** [review]
